app/controllers/api/articles_controller.rb

```diff
@@ -2,15 +2,25 @@ class Api::ArticlesController < ApplicationController
   def index
     if params[:lat] && params[:long]
       location = Geocoder.search([params[:lat], params[:long]])
-      raw_list = Article.where(location: location.first.city, article_type: params[:article_type]).sort_by(&:created_at).reverse
+      raw_list = Article.where(location: location.first.city).sort_by(&:created_at).reverse
       if raw_list == []
-        raw_list = Article.where(article_type: params[:article_type]).sort_by(&:created_at).reverse
+        raw_list = Article.all.sort_by(&:created_at).reverse
         render json: raw_list, each_serializer: ArticlesIndexSerializer,
                meta: "We found no local articles from #{location.first.city}.", meta_key: :message
       else
         render json: raw_list, each_serializer: ArticlesIndexSerializer,
-        meta: location.first.city, meta_key: :location
+               meta: "Latest articles from #{location.first.city}", meta_key: :message
```

Comment:
Be more precise with what is changed in which PR. 
This belongs to some kind of Hotfix PR